user/models.py

```diff
@@ -0,0 +1,85 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+from django.contrib.auth.models import User
+from item.models import Item, Currency
+from user.managers import UserWalletManager, UserWatchlistManager
+
+
+class Watchlist(models.Model):
+    """Current user, favorite list of stocks"""
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name='watchlists'
+    )
+    item = models.ForeignKey(
+        Item,
+        blank=True,
+        null=True,
+        on_delete=models.CASCADE,
+        related_name='watch_list'
```

Comment:
Try to keep same naming conventions for related names, for example plural and without underscores `watchlists`